src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -313,6 +355,18 @@ void pychip_DeviceController_SetLogFilter(uint8_t category)
 #endif
 }
 
+ChipError::StorageType pychip_BLEMgrImpl_ConfigureBle(uint32_t bluetoothAdapterId)
```

Comment:
why does ChipDeviceController* cpp file contain a "BLEMgrImpl" method?

Reply:
There are other BLE methods in this file that do eventually, go through to the platform layer. Any suggestions?

Reply:
Probably not in the scope of this pr, resolving  this.

I am expressing some disappointment/concern here: we seem to start lumping all functioanlity in one file ("ScriptBinding") because it is convenient to write so we sacrifice modularity and long term maintenance for faster developer writing. Maybe that is a tradeoff we can do in the short term for v1.0. However the correct thing to do here at least is for all `pychip_BLEMgrImpl` to have their own `py-blemgr.cpp` or similar file (specifically I would not name them script binding for example - these are already in a python directory ... I am also complaining that they live under src/controller ... blemgr has nothing to do with controller)

Reply:
Feel free to resolve this comment as a noop. I did not resolve it right away to just have it read and we can discuss outside of this PR if there is anything to be done. This comment should not be considered as blocking the PR for sure.